pkgs/tools/networking/octodns/providers/hetzner/default.nix

```
@@ -0,0 +1,47 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, octodns
+, pythonOlder
+, requests
+, setuptools
+, wheel
+}:
+
+buildPythonPackage rec {
+  pname = "octodns-hetzner";
+  # the latest release tag is over a year behind.
+  version = "unstable-2023-09-29";
```

```suggestion
  version = "0.0.2-unstable-2023-09-29";
```
It may be helpful to add the latest released version to see what version it is based on.
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming